pandas-stubs/core/frame.pyi

```diff
@@ -104,15 +108,26 @@ class DataFrame:
     def drop(self, index: Union[List[_str], Index], axis: _AxisType = ...) -> DataFrame: ...
     def drop_duplicates(self, keep: Union[_str, bool] = ...) -> DataFrame: ...
     def dropna(self, axis: int = ..., how: _str = ...) -> DataFrame: ...
+    @overload
+    def fillna(
+        self,
+        value: Union[float, Dict, Series, DataFrame] = ...,
+        method: _str = ...,
+        axis: Union[_str, int] = ...,
+        inplace: Literal[False] = ...,
+        limit: int = ...,
+        downcast: Dict = ...,
+    ) -> DataFrame: ...
+    @overload
     def fillna(
         self,
+        inplace: Literal[True],
```

Comment:
Did you need to move this line to make it work? It destroys the symmetry a bit :)

Reply:
Agreed, it looks weird; however, moving it to the right place throws an error: 'non-default argument follows default argument'

Reply:
oh, of course. yeah, makes sense.